modules/hitlnext/src/backend/webhook.ts

```diff
@@ -15,7 +15,7 @@ class WebHookService {
 
   private async setup() {
     this.config = (await this.bp.config.getModuleConfig(MODULE_NAME)).eventsWebHook
-    this.disabled = !this.config
+    this.disabled = !(this.config && this.config.url)
```

Comment:
Personal Preference (PP):

```suggestion
    this.disabled = !(this.config?.url)
```